example/sandbox.cpp

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
OOps sandbox.cpp slipped in, just a work in progress source. Will remove it shortly